forge-game/src/main/java/forge/game/ability/effects/EndTurnEffect.java

```diff
@@ -30,7 +33,13 @@ public void resolve(SpellAbility sa) {
         // 1) All spells and abilities on the stack are exiled. This includes
         // Time Stop, though it will continue to resolve. It also includes
         // spells and abilities that can't be countered.
-        game.getAction().exile(new CardCollection(game.getStackZone().getCards()), sa, null);
+        Map<AbilityKey, Object> moveParams = AbilityKey.newMap();
+        CardZoneTable table = new CardZoneTable(sa.getLastStateBattlefield(), sa.getLastStateGraveyard());
+        AbilityKey.addCardZoneTableParams(moveParams, table);
+
+        game.getAction().exile(new CardCollection(game.getStackZone().getCards()), sa, moveParams);
+        // trigger of Table not needed?
```

Comment:
> 721.1f. No player gets priority during this process, so triggered abilities are not put onto the stack. If any triggered abilities have triggered since this process began, those abilities are put onto the stack during the cleanup step, then the active player gets priority and players can cast spells and activate abilities.

Imo _Stonebinder's Familiar_ should trigger